internal/service/lightsail/container_service.go

```diff
@@ -0,0 +1,997 @@
+package lightsail
+
+import (
+	"context"
+	"log"
+	"reflect"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/lightsail"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	tftags "github.com/hashicorp/terraform-provider-aws/internal/tags"
+	"github.com/hashicorp/terraform-provider-aws/internal/verify"
+)
+
+func ResourceContainerService() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceContainerServiceCreate,
+		ReadContext:   resourceContainerServiceRead,
+		UpdateContext: resourceContainerServiceUpdate,
+		DeleteContext: resourceContainerServiceDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		CustomizeDiff: verify.SetTagsDiff,
+
+		Schema: map[string]*schema.Schema{
+			// required fields
```

Comment:
Schema fields should be defined in alphabetical order, not grouped by required/not required